core/dbt/parser/models.py

```diff
@@ -205,16 +205,16 @@ def parse_python_model(self, node, config, context):
 
         dbtParser = PythonParseVisitor(node)
         dbtParser.visit(tree)
-
+        config_get_keys = []
```

Comment:
Not a blocker, but `config_get_keys` is hard to reason about for me. `config_keys_used` is a little easier to understand, if that is what this is meant to hold.

Reply:
Thanks!! It's def easier to understand! Updated